### DISCUSSION TITLE
Cannot Sign in to FIreFly

### DISCUSSION THREAD

**Discussion by lonesoac0** [firefly-iii/firefly-iii#6341] [General]
Hello all,

    I have installed YunoHost and then I have installed FireFly as an app within YunoHost.  I cannot sign into the FireFly application with the admin account I created for FireFly.  I have reset the password to the account in the YunoHost system as well.  What am I missing?

Thank you.

**Comment by JC5** [comment]
I'm not sure what the problem is. Were you able to remove Firefly III and reinstall it?

**Comment by lonesoac0** [comment]
Yes, and the same thing happened.

**Comment by JC5** [comment]
Have you reset the password of the Firefly III account?
